tracer/src/Datadog.Trace/AppSec/AsmRemoteConfigurationProducts.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="AsmRemoteConfigurationProducts.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.AppSec;
+
+internal class AsmRemoteConfigurationProducts
```

Comment:
nit: could make this static too

Reply:
Good point. I'll make the change in a future PR to avoid a new CI run.